playbooks/chameleon_usage.yml

```diff
@@ -1,4 +1,26 @@
 ---
 - hosts: chameleon_usage
   roles:
-    - chameleon_usage
\ No newline at end of file
+    - role: chameleon_usage
+      vars:
+          action: 'mysql_init'
+    - role: chameleon_usage
+      vars:
+          action: 'config'
+    - role: chameleon_usage
+      vars:
+          action: 'node'
+    - role: chameleon_usage
+      when: enable_usage_appliance_report
+      vars:
+          action: 'heat_template'
+
+- hosts: chameleon_usage_jupyter
+  roles:
+    - role: chameleon_usage
+      vars:
+          action: 'config'
+    - role: chameleon_usage
+      when: enable_usage_jupyter_report
```

Comment:
Is it the case that there would be a host marked as `chameleon_usage_jupyter` but you wouldn't want `enable_usage_jupyter_report`? Perhaps we don't need this variable.